src/runtime/auto-prompt-manager-test.js

```diff
@@ -1789,10 +1814,10 @@ describes.realWin('AutoPromptManager', (env) => {
 
     it('With SurveyTriggeringPriorityExperiment enabled, should show the Survey prompt before any actions', async () => {
       setupPreviousImpressionAndDismissals(storageMock, {
-        dismissedPromptGetCallCount: 1,
+        dismissedPromptGetCallCount: 2,
         getUserToken: true,
       });
-      miniPromptApiMock.expects('create').once();
+      miniPromptApiMock.expects('create').never();
 
       await autoPromptManager.showAutoPrompt({
         autoPromptType: AutoPromptType.CONTRIBUTION,
```

Comment:
(FYI, Not actionable)
AFAIK, the "mini" contribution UI has been deprecated, and now AutoPromptType is set as `CONTRIBUTION_LARGE` by default in the code snippet provided in publisher center. So this edge case (for contribution) should not exist in practice.

Reply:
+1, created b/278874256 to cleanup miniPromptApi.